drulenium_visual_testing/README.md

```diff
@@ -0,0 +1,47 @@
+# Visual Regression Testing via Drulenium #
+
+This example will show you how you can automatically run visual tests on your pantheon hosted environments using Drulenium suite of modules & libraries. 
+
+This will allow you to do a comparative visual diff between the test environment and the development environemnt(including Multidev environments) everytime you push code to the development environment.
+
+As a note, Tests run on https://travis-ci.org/ and is free to use for unlimited number of pages but the test results are public. For more information read http://drulenium.org/how-to-automate-testing-in-pantheon-hosted-website
+
+## Instructions ##
+
+Setting up this example is easy:
+
+1. Add the example post_drulenium_github.php script to the 'private/scripts/' directory of your code repository.
+2. Add the secret github account parameters into a file called secrets.json and store it in the private files directory of Test & Live environments and Clone files from the test environment into other dev environments.
```

Comment:
@ndubbaka Can you specify the location of the private directory on the test/live environments? It should be clear to create `secrets.json` a user must connect to each environment via FTP and edit the file `files/private/secrets.json`.

I just don't want people to get the environment private directory confused with the private directory in the Git root.